CMakeExternals/DCMTK.cmake

```diff
@@ -24,7 +24,7 @@ if(DEFINED DCMTK_DIR AND NOT EXISTS ${DCMTK_DIR})
 endif()
 
 if(NOT DEFINED DCMTK_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
-  set(revision_tag DCMTK-3.6.1_20150924)
+  set(revision_tag efe1a177eac040b6d41aae0db7714cd970e12bc5)
```

Comment:
Could you add a comment specifying what is this version ? Is it the latest nightly that was available ?